### ISSUE TITLE
Uploading multiple assets fail

### ISSUE THREAD

**Issue by pcoltau** [BreadMoirai/github-release-gradle-plugin#21]
I'm using version 2.2.8 of this plugin and gradle version 5.4.

My configuration: 

```
githubRelease {
    token githubToken
    draft false // by default this is false
    prerelease false // by default this is false
    releaseAssets jar.destinationDirectory.asFile.get().listFiles() // this points to which files you want to upload as assets with your release
}
```

When running `./gradlew githubRelease`, I get the following issue:

```
> Task :githubRelease
:githubRelease CHECKING FOR PREVIOUS RELEASE
:githubRelease CREATING NEW RELEASE
:githubRelease STATUS CREATED
:githubRelease https://github.com/[redacted]/releases/tag/v0.0.1
:githubRelease UPLOADING ASSETS
:githubRelease UPLOADING [redacted]-0.0.1.jar.asc
:githubRelease UPLOADING [redacted]-0.0.1.jar
:githubRelease FAILED TO UPLOAD [redacted]-0.0.1.jar
422 Unprocessable Entity
[message:Validation Failed, request_id:E8B3:75CB:3545C7:3B90D2:5CC1B174, documentation_url:https://developer.github.com/v3, errors:[[resource:ReleaseAsset, code:already_exists, field:name]]]
:githubRelease UPLOADING [redacted]-0.0.1-javadoc.jar
:githubRelease FAILED TO UPLOAD [redacted]-0.0.1-javadoc.jar
422 Unprocessable Entity
[message:Validation Failed, request_id:E8B3:75CB:3545CA:3B90DB:5CC1B175, documentation_url:https://developer.github.com/v3, errors:[[resource:ReleaseAsset, code:already_exists, field:name]]]
:githubRelease UPLOADING [redacted]-0.0.1-javadoc.jar.asc
:githubRelease FAILED TO UPLOAD [redacted]-0.0.1-javadoc.jar.asc
422 Unprocessable Entity
[message:Validation Failed, request_id:E8B4:14D8:33488A:3965A8:5CC1B175, documentation_url:https://developer.github.com/v3, errors:[[resource:ReleaseAsset, code:already_exists, field:name]]]
:githubRelease UPLOADING [redacted]-0.0.1-sources.jar
:githubRelease FAILED TO UPLOAD [redacted]-0.0.1-sources.jar
422 Unprocessable Entity
[message:Validation Failed, request_id:E8B4:14D8:334891:3965AE:5CC1B176, documentation_url:https://developer.github.com/v3, errors:[[resource:ReleaseAsset, code:already_exists, field:name]]]
:githubRelease UPLOADING [redacted]-0.0.1-sources.jar.asc
:githubRelease FAILED TO UPLOAD [redacted]-0.0.1-sources.jar.asc
422 Unprocessable Entity
[message:Validation Failed, request_id:E8B4:14D8:334898:3965B9:5CC1B176, documentation_url:https://developer.github.com/v3, errors:[[resource:ReleaseAsset, code:already_exists, field:name]]]
```

**Comment by BreadMoirai** [comment]
Hello, sorry for the late fix, this should be resolved in version 2.2.9

**Comment by finngaida** [comment]
I'm afraid this issue has reappeared with 2.2.12
